nl/guides/for-students/README.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Van start met Dodona voor studenten
```

Comment:
This page has 3 different titles: "for-students" as file name, "dodona voor studenten" as h1 on the page and "van start met Dodona voor studenten" as navigation title. At least the last 2 should be more or less the same. I think "Van start met Dodona als student" is a good title, but then the other "van start met dodona" page should be renamed similarly.